velodyne_pointcloud/src/lib/rawdata.cpp

```diff
@@ -384,21 +450,164 @@ void RawData::unpack(
         intensity = (intensity < min_intensity) ? min_intensity : intensity;
         intensity = (intensity > max_intensity) ? max_intensity : intensity;
 
-        float time = 0;
-        if (timing_offsets_.size()) {
-          time = timing_offsets_[i][j] + time_diff_start_to_this_packet;
-        }
-
```

Comment:
If this is all going to get skipped for the 128, why remove this code?

Reply:
it is not removed it moved up to line 336 to 339, because of line 341 to 346. (also ee the comment there.